.travis.yml

```diff
@@ -8,19 +8,33 @@ cache:
 matrix:
   include:
     - node_js: "8"
-      env: PERCY_ENABLE=1
-    - node_js: "7"
-      env: PERCY_ENABLE=0
-    - node_js: "6"
-      env: PERCY_ENABLE=0
+      env: LINT=true
     - node_js: "4"
-      env: PERCY_ENABLE=0
+      env: UNIT_TEST=true COVERAGE=false
+    - node_js: "6"
+      env: UNIT_TEST=true COVERAGE=false
+    - node_js: "7"
+      env: UNIT_TEST=true COVERAGE=false
+    - node_js: "8"
+      env: UNIT_TEST=true COVERAGE=true
```

Comment:
Run unit tests on each Node version, but only generate the coverage report on latest since it slows down the test run a bit.